### DISCUSSION TITLE
Bosch RA II + M - Paring not working

### DISCUSSION THREAD

**Discussion by titro** [Koenkk/zigbee2mqtt#25036] [Devices]
Hey folks,

i got a Bosch RA II +M today which should support Zigbee + Matter over Thread.

I have a running zigbee2mqtt with a Sonoff Zigbee Stick and Bosch Raumthermostat II (without M) is working fine.

The new RA +M only has a Matter QR Code so i generated the correct one for Zigbee like described in the doc:

https://www.zigbee2mqtt.io/devices/BTH-RA.html

But this is not working. RA is in Pairing mode (i already resetted it several times) and nothing is happened after entering the install code in zigbee2mqtt tools sections (which was working flawless with the "normal" Bosch RA II)

Here are the codes:
![image](https://github.com/user-attachments/assets/9ab6731e-6b88-4632-bbf3-d0fa10382bf8)

And the manual generated install code like described in the zigbeemqtt doc should be
RB01SG0D83101826480080000000000000000000BC026EFFFE8D50E2DLK5E835FA1FF3E7221783039303A14B87B5B3D

Any hints or ideas?

thanks!

**Comment by TissenMi** [comment]
Thanks for opening this Post… 
I have the same. After giving the install-code, join appears, but nothing happens and the Thermostat is Not installed. 

Thanks. Michael

**Reply by titro** [reply]
Got the solution. RA is in thread Mode at the beginning (the Symbol m is visible on the Display)
Press and hold the Button until the led Ring is Orange and a z is visible. After that pairing is possible with zigbee.

**Reply by TissenMi** [reply]
Thanks for the Tipp and so it works. When you press the Button for Reset und M appears, press the Button twice and it changed to Z. 

So i have another Question. I have paired it whis Matter to my Apple TV. How can I delete this. When I press the Button to Reset, there is no M in the Display and I cannot change to anything.

**Reply by titro** [reply]
To reset the whole RA :

Remove one of the two battery.
Press and hold the Button while inserting the second battery until remove is visible. Then press once to confirm

**Reply by TissenMi** [reply]
Yeah… Thanks allot!! 

And is right, that I can pair it in Matter OR in Zigbee, not in both?

**Reply by titro** [reply]
Yesssss
